plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -5057,19 +5057,27 @@ public void testSplitPruningForFilterOnNonPartitionColumn()
     public void testGetIcebergTableProperties()
     {
         assertUpdate("CREATE TABLE test_iceberg_get_table_props (x BIGINT)");
-        verifyIcebergTableProperties(computeActual("SELECT * FROM \"test_iceberg_get_table_props$properties\""));
+        verifyIcebergTableProperties(computeActual("SELECT * FROM \"test_iceberg_get_table_props$properties\""), HiveCompressionCodec.ZSTD);
         assertUpdate("DROP TABLE test_iceberg_get_table_props");
     }
 
-    protected void verifyIcebergTableProperties(MaterializedResult actual)
+    protected void verifyIcebergTableProperties(MaterializedResult actual, HiveCompressionCodec codec)
     {
         assertThat(actual).isNotNull();
-        MaterializedResult expected = resultBuilder(getSession())
+        Optional<String> codecName = switch (format) {
+            case AVRO -> IcebergUtil.toAvroCompressionCodecTableProperty(codec);
```

Comment:
We're using for testing the same code as the production code. I don't think whether this is the way to spot eventual regressions. Why not rather explicitly hardcode the codec name ?

Reply:
Good call - I will inline this to strings - makes it also easier on the eyes.